TShockAPI/Bouncer.cs

```diff
@@ -2496,5 +2511,45 @@ internal static int GetMaxPlaceStyle(int tileID)
 			{ ProjectileID.GoldShortswordStab, ItemID.GoldShortsword },
 			{ ProjectileID.PlatinumShortswordStab, ItemID.PlatinumShortsword }
 		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI = new Dictionary<short, short> {
+			{ 611, -1 },
+
+			{ 950, 0 }
+		};
+		private Dictionary<short, short> Projectile_MaxValuesAI = new Dictionary<short, short> {
+			{ 611, 1 },
+
+			{ 950, 0 }
+		};
+
+		private Dictionary<short, short> Projectile_MinValuesAI2 = new Dictionary<short, short> {
+			{ 405, 0 },
+			{ 410, 0 },
+
+			{ 424, 0 },
+			{ 425, 0 },
+			{ 426, 0 },
+
+			{ 612, 0 },
```

Comment:
```suggestion
			{ 612, 0.4f },
```
Source: `Player.ItemCheck_Shoot()`, `float ai4 = (Main.rand.NextFloat() - 0.5f) * ((float)Math.PI / 4f);`